deploy/kubernetes/base/node-daemonset.yaml

```diff
@@ -46,7 +46,7 @@ spec:
           args:
             - --endpoint=$(CSI_ENDPOINT)
             - --logtostderr
-            - --v=2
+            - --v=5
```

Comment:
we reduced default log level to 2 last year as we think this is bit too high. 

These messages are set at level 4/5 which is meant for debugging and does not need to be set to 5

Reply:
Can we change this back to 2? Also can we squash  and only keep the crucial commits?